src/openrct2-ui/windows/Window.h

```diff
@@ -11,6 +11,7 @@
 
 #include <openrct2-ui/interface/Window.h>
 #include <openrct2/Identifiers.h>
+#include <openrct2/ride/RideConstruction.h>
```

Comment:
Rather than including this header in this common file, can we make do with a forward declaration of the type(s) required?

Reply:
I’ll try it.